src/settings.py

```diff
@@ -117,8 +117,8 @@
     ("fr", _("French")),
     ("en", _("English")),
 ]
-LOCALE_PATHS = ["/sylvis/locale"]
```

Comment:
why removing this? I thought it was mandatory.

Reply:
That setting is for additional paths (the `locale` folder under installed apps is already recognized, see https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#how-django-discovers-translations)